charts/alfresco-activiti-admin/values.yaml

```
@@ -82,6 +82,12 @@ database:
       # -- Key within the secret holding the database password for admin app
       password: DATABASE_PASSWORD
 
+workaround:
+  # -- Enable workaround for https://hyland.atlassian.net/browse/ACTIVITI-5113
+  # Set to `true` to enable the workaround regardless of the appVersion
```

```suggestion
  # Set to `true` to enable the workaround
```

why mention appVersion if there is not logic around it?

---

Leftovers! Thank you/